### PR TITLE
Release v1.3.1

### PR DESCRIPTION
**Pull request by carlosmonteagudoinbibo**
In this PR we update the documentation for patch v1.3.1, which includes:
- write the section in release notes
- small updates in FAQ
- small updates in Licensing 

### Release notes
image

### FAQ
image

### Licensing
image

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -106,10 +106,10 @@ No, floating licensing requires internet access to activate the server. To be mo
...
+You can use the environment variable `ADN_LICENSE_MODE` set to `0` for node-locked licensing and `ADN_LICENSE_MODE` set to `1` for floating licensing. The license gets verified only once when the plug-in is loaded. This means that if a change like this one needs to be applied, then the application has to be restarted to load the plug-in again from a new process.
 
 ## What do I need to be able to use floating licensing?
-You will need to configure and activate the licensing server, make sure that client machines have access to it.
+First of all, the licensing server has to be installed, configured and activated previously (check this [section](licensing#floating-licensing)). Then, the client machine(s) on which AdonisFX will be used have to be configured to allow them to connect to the server to request leases. The configuration simply requires to set a few environment variables: `ADN_LICENSE_MODE` to `1`; and `ADN_LICENSE_SERVER` and `ADN_LICENSE_SERVER_BATCH` to the IP address and port (e.g. `127.0.0.1:13`) of the interactive and floating server respectively.
```
Comment (lucasferryinbibo) [inline docs/faq.md:112]: I think you meant this right?
```suggestion
First of all, the licensing server has to be installed, configured and activated previously (check this [section](licensing#floating-licensing)). Then, the client machine(s) on which AdonisFX will be used have to be configured to allow them to connect to the server to request leases. The configuration simply requires to set a few environment variables: `ADN_LICENSE_MODE` to `1`; and `ADN_LICENSE_SERVER` and `ADN_LICENSE_SERVER_BATCH` to the IP address and port (e.g. `127.0.0.1:13`) of the interactive and batch floating server respectively.
```

**Review by lucasferryinbibo — APPROVED** [review]
One small comment!

File: docs/faq.md (hunk 1 above)
Comment (carlosmonteagudoinbibo) [reply docs/faq.md]: Fixed, thanks for spotting it

**Review by josevalverdeinbibo — APPROVED** [review]
Approved!

**Pull request merged**